modules/tools/cluster-autoscaler/cluster-autoscaler-values.tpl

```diff
@@ -1,7 +1,11 @@
+image:
+  repository: k8s.gcr.io/autoscaling/cluster-autoscaler
+  tag: v1.21.0
```

Comment:
the AWS docs for cluster autoscaler _claim_ that you're supposed to use a tag that reflects what version of k8s you're running (so in this case, `v1.17.0`), but I don't believe that to be true.  this configuration seems to work, but I want to test it out a bit more before we merge this